buildenv/jenkins/jobs/infrastructure/Cleanup-Nodes

```diff
@@ -232,6 +236,18 @@ timeout(time: TIMEOUT_TIME.toInteger(), unit: TIMEOUT_UNITS) {
     }
 }
 
+/*
+* Return a list of workspace directories (current build workspace excluded)
+*/
+def get_other_workspaces(workspaceDir) {
+    // fetch all directories in workspaceDir (this should not fail)
+    def workspaces = sh(script: "ls ${workspaceDir}", returnStdout: true).trim().tokenize(System.getProperty('line.separator'))
```

Comment:
`System.getProperty('line.separator'))` -> `System.lineSeparator()`

More efficient to call the method that directly returns the value rather than doing a property lookup on each call

Reply:
good point, replaced